pdc_client/plugins/build_images.py

```diff
@@ -39,7 +39,9 @@ def register(self):
     def _print_build_image_list(self, build_images, with_md5=False):
         fmt = '{image_id}'
         if with_md5:
-            fmt += ' {md5}'
+            fmt = '{image_id:50}{md5}'
+        print fmt.format(image_id='Image ID', md5='MD5')
+        print
```

Comment:
For me it looks more clear. It also displayed like this way in compose-image-rtt-tests, compose-tree-locations. Not sure if this way is real better. @simozhan what do you think?